src/primitive.js

```diff
@@ -183,7 +183,7 @@ var _ = Mavo.Primitive = class Primitive extends Mavo.Node {
 		if (this.editor) {
 			if (_.isFormControl(this.editor)) {
 				if (this.editor.matches("select")) {
-					let text = [...this.editor.options].find(o => o.value == value)?.textContent;
+					let text = [...this.editor.options].find(o => (Array.isArray(value)? value : [value]).map(v => v + "").includes(o.value))?.textContent;
```

Comment:
Isn't `Array.isArray(value)? value : [value]` basically `Mavo.toArray()`? (IIRC)

`v + ""` will fail badly if array items are objects, is that a possibility?

Reply:
> Isn't Array.isArray(value)? value : [value] basically Mavo.toArray()? (IIRC)

You are absolutely right! Thanks. Somehow I forgot about this utility method.

Reply:
> v + "" will fail badly if array items are objects, is that a possibility?

Hm. I don’t know. I need to investigate it.

Reply:
> v + "" will fail badly if array items are objects

Will the `toString()` method do in that case? It seems to me, it covers all the cases we might face.

Reply:
No, `toString()` does exactly the same as `v + ""`. E.g. `({foo: 2}).toString()` is `'[object Object]'`, which is not really helpful.